src/core/main.c

```diff
@@ -1364,7 +1363,6 @@ int main(int argc, char *argv[]) {
 
                 /* Running outside of a container as PID 1 */
                 arg_system = true;
-                make_null_stdio();
```

Comment:
I'm not sure why you're mentioning "services", here. I'm now confused.

Are there any useful cases where a systemd process (being started with --system or --user and/or from a container) has its stdio not connected to /dev/null ?